locale/en/foreman_remote_execution.po

```diff
@@ -1,19 +1,760 @@
+
+msgid "Duplicated inputs detected: %{duplicated_inputs}"
+msgstr ""
+
+msgid "This template is locked. Please clone it to a new template to customize."
+msgstr ""
+
+msgid ""
+"Search the host for any proxy with Remote Execution, useful when the host has "
+"no subnet or the subnet does not have an execution proxy"
+msgstr ""
+
+msgid ""
+"Search for remote execution proxy outside of the proxies assigned to the host."
+" If locations or organizations are enabled, the search will be limited to the "
+"host's organization or location."
+msgstr ""
+
+msgid ""
+"Default user to use for SSH.  You may override per host by setting a parameter"
+" called remote_execution_ssh_user."
+msgstr ""
+
+msgid ""
+"Default user to use for executing the script. If the user differs from the SSH"
+" user, su or sudo is used to switch the user."
+msgstr ""
+
+msgid "What command should be used to switch to the effective user. One of %s"
+msgstr ""
+
+msgid "SSH"
+msgstr ""
+
+msgid "Effective user method \"%{current_value}\" is not one of %{valid_methods}"
+msgstr ""
+
+msgid "Static Query"
+msgstr ""
+
+msgid "Dynamic Query"
+msgstr ""
+
+msgid "Cannot resolve hosts without a user"
+msgstr ""
+
+msgid "Cannot resolve hosts without a bookmark or search query"
+msgstr ""
+
+msgid "Must select a bookmark or enter a search query"
+msgstr ""
+
+msgid "User input"
+msgstr ""
+
+msgid "Fact value"
+msgstr ""
+
+msgid "Variable"
+msgstr ""
+
+msgid "Puppet parameter"
+msgstr ""
+
+msgid "Value for required input '%s' was not specified"
+msgstr ""
+
+msgid "Input"
+msgstr ""
+
+msgid "Not all required inputs have values. Missing inputs: %s"
+msgstr ""
+
+msgid "The execution interface is used for remote execution"
+msgstr ""
+
+msgid "Remote execution"
+msgstr ""
+
+msgid "Remote Execution"
+msgstr ""
+
+msgid "Proxies"
+msgstr ""
+
+msgid ""
+"Select as many remote execution proxies as applicable for this subnet.  When m"
+"ultiple proxies with the same provider are added, actions will be load balance"
+"d among them."
+msgstr ""
+
+msgid "Description"
+msgstr ""
+
+msgid "Job template"
+msgstr ""
+
+msgid "Resolves to"
+msgstr ""
+
+msgid "hosts"
+msgstr ""
+
+msgid "Preview"
+msgstr ""
+
+msgid "Display advanced fields"
+msgstr ""
+
+msgid "Hide advanced fields"
+msgstr ""
+
+msgid "Effective user"
+msgstr ""
+
+msgid ""
+"A user to be used for executing the script. If it differs from the SSH user, s"
+"u or sudo is used to switch the accounts."
+msgstr ""
+
+msgid "Type of query"
+msgstr ""
+
+msgid ""
+"Type has impact on when is the query evaulated to hosts.<br><ul><li><b>Static<"
+"/b> - evaluates just after you submit this form</li><li><b>Dynamic</b> - evalu"
+"ates just before the execution is started, so if it's planed in future, target"
+"ed hosts set may change before it</li></ul>"
+msgstr ""
+
+msgid ""
+"The final host list may change because the selected query is dynamic.  It will"
+" be rerun during execution."
+msgstr ""
+
+msgid "...and %{count} more"
```

Comment:
This should probably use `n_()` for pluralisation, even if it's the same in English.